zabbix-monitoring-cluster/azuredeploy.json

```diff
@@ -0,0 +1,169 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "clusterResourcePrefix": {
+      "type": "string",
+      "metadata": {
+        "description": "Prefix for cluster resources."
```

Comment:
since this forms the base of the public IP FQDN, should mention in the descripiton that it needs to be unique across all of Azure (not just your resources).